bootstrap.py

```diff
@@ -348,8 +348,12 @@ def migrate_systems(org_name, activationkey):
     else:
         options.rhsmargs += " --keep"
     exec_failok("/usr/sbin/subscription-manager config --server.server_timeout=%s" % options.timeout)
-    exec_failexit("/usr/sbin/rhn-migrate-classic-to-rhsm --org %s --activation-key '%s' %s" % (org_label, activationkey, options.rhsmargs))
-    exec_failexit("subscription-manager config --rhsm.baseurl=https://%s/pulp/repos" % options.foreman_fqdn)
+    if options.ignore_registration_failures:
```

Comment:
reading these branches, I wonder how often we will forget to update params in one of them.

how about we mangle `exec_failok` and `exec_failexit` into *one* function `exec(command, failok=False)` and make the other two sensible aliases to it? then we can just pass `ignore_r_f` as the `failok` param.

Reply:
ack, but I'd rather handle a refactoring of the `exec*` functions as a different PR. (because I am bound to mess it up :) )